### PR TITLE
Update NG and use newer installer versions

### LINKED ISSUE #209: Universal JAR is always packed inside the installer JAR
Currently, the installer JAR is packed with the universal JAR (under the `maven/` folder inside the installer), despite the universal JAR also being published and downloadable from our Maven repository.

During NeoForge development, it's helpful to have the universal JAR packed with the installer as to not require publishing the universal JAR to e.g. the local Maven repository (`~/.m2`). Outside of development though, there's little benefit to packing the universal JAR, since the installer will automatically download it alongside all other libraries.

The packing of the universal JAR is controlled by [the `neogradle.runtime.platform.installer.debug` project property in NeoGradle](https://github.com/neoforged/NeoGradle/blob/2442ea835b255855be7ca3df9318b88ca0590214/platform/src/main/java/net/neoforged/gradle/platform/extensions/DynamicProjectExtension.java#L491-L496), which is [currently set to `true` in NeoForge](https://github.com/neoforged/NeoForge/blob/14ee4f7e9aefaec86d2189fc1c5367b830256f67/gradle.properties#L52C1-L52C43). The property should be set to `true` only in a local/non-CI environment.

### PR DESCRIPTION
**Pull request by Matyrobbrt**
TL;DR of the installer changes:
- the radio buttons no longer clip, the version is in bold
- the profile name is now `NeoForge` not `neoforge`
- the neo logo is now 80% of the size, since it was just way too big
- the installer now has window icons
- add a CLI option to install a client profile
- add a second progress bar that indicates the status of the current step; that is, the progress of the downloading of the server jar, the amount of libraries downloaded out of the total lib count, the progress of each step executed by the installer (download mappings, deobf, binpatch)

Also, no longer pack the universal installer jar in the CI-built installers.

Fixes #209

### CODE REVIEW

**Comment by marchermans** [comment]
I would request a change in the CI and properties:

Instead of disabling the debug on CI only, lets disable it by default. It was never mend to be true by default.

**Comment by Matyrobbrt** [comment]
No, it makes sense for it to be enabled. It's useful for any local changes since you don't have to publish to maven local too (and it's portable).

**Comment by marchermans** [comment]
Oke

**Review by marchermans — APPROVED** [review]
